packages/front-end/components/Experiment/EditMetricsForm.tsx

```diff
@@ -79,7 +70,7 @@ export function fixMetricOverridesBeforeSaving(overrides: MetricOverride[]) {
       if (key === "id") continue;
       const v = overrides[i][key];
       // remove nullish values from payload
-      if (v === undefined || v === null || isNaN(v)) {
+      if (v === undefined || v === null || (key !== "windowType" && isNaN(v))) {
```

Comment:
Instead of this, we can just change `isNaN` to `Number.isNaN` for a more robust and maintainable check.

`isNaN` will return false for anything that is not a number.  Whereas `Number.isNaN` only checks for the literal `NaN`.